.github/workflows/static.yml

```diff
@@ -10,7 +10,6 @@ jobs:
 
     env:
       extensions: ast
```

Comment:
Not required for psalm

Reply:
Can you fix in [template](https://github.com/yiisoft/template/blob/master/.github/workflows/static.yml) ?
I'll move the actual config here later.

Reply:
[Fixed](https://github.com/yiisoft/template/commit/7d77f1512415e96e008af36f8cf8352a1c039cfd)

Reply:
Done!